content/src/Server.ts

```diff
@@ -47,12 +50,18 @@ export class Server {
     const upload = multer({ dest: Server.UPLOADS_DIRECTORY, preservePath: true })
     const controller: Controller = env.getBean(Bean.CONTROLLER)
     this.garbageCollectionManager = env.getBean(Bean.GARBAGE_COLLECTION_MANAGER)
-    this.synchronizationManager = env.getBean(Bean.SYNCHRONIZATION_MANAGER)
     this.snapshotManager = env.getBean(Bean.SNAPSHOT_MANAGER)
     this.service = env.getBean(Bean.SERVICE)
     this.migrationManager = env.getBean(Bean.MIGRATION_MANAGER)
     this.repository = env.getBean(Bean.REPOSITORY)
 
+    if (env.getConfig(EnvironmentConfig.DISABLE_SYNCHRONIZATION)) {
+      Server.LOGGER.warn(`Cluster synchronization has been disabled.`)
+      this.synchronizationManager = new DisabledSynchronizationManager()
```

Comment:
I could just modified the bean for that  case, but it seem more clear just to have an env var 🤔